src/Analyzers/CSharp/Analyzers/UseIndexOrRangeOperator/CSharpUseIndexOperatorDiagnosticAnalyzer.cs

```diff
@@ -194,7 +195,10 @@ protected override void InitializeWorker(AnalysisContext context)
                 return;
             }
 
-            if (CSharpSemanticFacts.Instance.IsInExpressionTree(instance.SemanticModel, instance.Syntax, infoCache.ExpressionOfTType, cancellationToken))
+            var semanticModel = instance.SemanticModel;
+            Contract.ThrowIfNull(semanticModel);
```

Comment:
NRT was turned on for a bunch of files.  so there's ancillary fallout of htat in this PR.

Reply:
Lgtm besides the rest of the null references that need fixing.